default/stringtables/en.txt

```diff
@@ -11519,15 +11519,16 @@ PLC_METROPOLES
 Metropoles
 
 PLC_METROPOLES_DESC
-'''Boosts the output of up to [[value METROPOLES_PLANET_LIMIT]] planets with population of at least [[value METROPOLES_MINIMUM_POPULATION]]. The highest-[[metertype METER_POPULATION]] of the qualified planets will receive, regardless of focus:
-• Increases [[metertype METER_TARGET_RESEARCH]] by [[value METROPOLES_TARGET_RESEARCH_PERPOP]] per [[metertype METER_POPULATION]].
+'''Boosts the output of up to [[value METROPOLES_PLANET_LIMIT]] planets (but no more than 1 / [[value METROPOLES_MIN_PLANETS]] or your populated planets) with [[metertype METER_POPULATION]] of at least [[value METROPOLES_MINIMUM_POPULATION]] and [[metertype METER_HAPPINESS]] of at least [[value METROPOLES_MIN_STABILITY]].
```

Comment:
don't refer to the player in the second person, eg. "your planets".

Reply:
Well, that sentence will probably change anyway. How you you write it? ...of all populated planets)

Reply:
refer to "of an empire" or "the empire's" or similar, rather than "your"